### ISSUE TITLE
Accessibility issue with email link

### ISSUE THREAD

**Issue by eddiekonczal** [joomla/joomla-cms#24448]
### Steps to reproduce the issue

- Click the email icon for an article
- Analyze the "Email this link to a friend" popup window to page with the "WorldSpace Attest DevTools" plugin for Google Chrome

### Expected result

The page should not return any accessibility errors

### Actual result

The following violation is reported:

> <html> element must have a valid value for the lang attribute
> issue1 of 1
> Issue description
> Ensures the lang attribute of the <html> element has a valid value
> 
> Impact: serious
>  Learn more
> Element location
> iframe[width="\33 04"] html
> Element source
> <html dir="ltr" lang="en_gb">
> To solve this violation, you need to:
> Fix the following:
> Value of lang attribute not included in the list of valid languages

### System information (as much as possible)

PHP Built On | Linux www3 4.4.162-94.72-default #1 SMP Mon Nov 12 18:57:45 UTC 2018 (9de753f) x86_64
Database Type | mysql
Database Version | 5.5.39
Database Collation | utf8mb4_general_ci
Database Connection Collation | utf8mb4_general_ci
PHP Version | 7.0.7
Web Server | Apache
WebServer to PHP Interface | apache2handler
Joomla! Version | Joomla! 3.9.4 Stable [ Amani ] 12-March-2019 15:00 GMT
Joomla! Platform Version | Joomla Platform 13.1.0 Stable [ Curiosity ] 24-Apr-2013 00:00 GMT
User Agent | Mozilla/5.0 (Windows NT 10.0; WOW64; rv:52.0) Gecko/20100101 Firefox/52.0 SeaMonkey/2.49.4

**Comment by hardik-codes** [comment]
@eddiekonczal could you please post the relevant screen shots for reproducing the issue

**Comment by eddiekonczal** [comment]
Here are the screen shots:

1. Click "Email this link to a friend" icon (upper right of article):
![1-click-email-link](https://user-images.githubusercontent.com/35043141/55355489-723def80-5496-11e9-942a-b33036577072.png)

2. Pres F12 in Chrome to bring up Developer Tools:
![2-press-F12-in-Chrome](https://user-images.githubusercontent.com/35043141/55355582-bdf09900-5496-11e9-940c-5b9c0e2387db.png)

3. Click "WorldSpace" in the Developer Tools Menu (this plugin must be installed first):

![3-click-worldspace](https://user-images.githubusercontent.com/35043141/55355655-eb3d4700-5496-11e9-8f13-ce518287884c.png)

4. Click "Analyze" to the lower left:

![4-click-analyze](https://user-images.githubusercontent.com/35043141/55355720-1e7fd600-5497-11e9-9b0e-047bf2ca76fc.png)

5. The violation will be displayed under Developer Tools:

![5-view-violation](https://user-images.githubusercontent.com/35043141/55355795-4d964780-5497-11e9-9eed-54431b60f8bc.png)

**Comment by zwiastunsw** [comment]
This is rather an error in your template.
Correct atribute is: lang="en-gb".
And such an attribute should be displayed.

**Comment by brianteeman** [comment]
Please close. 

@zwiastunsw has given the correct answer. There is nothing to fix in Joomla as it correctly uses a valid attribute

**Comment by ghost** [comment]
Closed as stated above.
